services/app/src/components/base/PlayerCount.tsx

```diff
@@ -0,0 +1,31 @@
+import * as React from 'react';
+import {MAX_ADVENTURERS} from '../../Constants';
+import Picker from './Picker';
+
+interface Props extends React.Props<any> {
+  allPlayers: number;
+  localPlayers: number;
+  id?: string;
+  onChange: (localPlayers: number) => any;
+}
+
+export default class PlayerCount extends React.Component<Props, {}> {
+  private onDelta(delta: number) {
+    if (delta > 0 && this.props.allPlayers + delta > MAX_ADVENTURERS) {
+      return;
+    }
+    if (delta < 0 && this.props.localPlayers + delta < 1) {
+      return;
+    }
+    this.props.onChange(this.props.localPlayers + delta);
+  }
+
+  public render() {
+    return (
+      <Picker id={this.props.id} label="Players" value={this.props.localPlayers} onDelta={(i: number) => this.onDelta(i)}>
+        {(this.props.allPlayers > 1) ? 'The number of players.' : <div><strong>Solo play:</strong> Play as two adventurers with double the combat timer.</div>}
```

Comment:
Technically not just the combat timer, affects the decision timer too... Not quite sure how to say that concisely tho

Reply:
Removed "combat".